### ISSUE TITLE
Plugin ignores css selector specificity

### ISSUE THREAD

**Issue by nseni** [jhildenbiddle/css-vars-ponyfill#112]
Hello. I found a bug with :root selector specificity.

Everything works fine with css classes: https://codepen.io/nseni/pen/bGdeoXv
```css
:root { --main-color: red; }
:root.theme-a { --main-color: green; }
:root.theme-b { --main-color: blue; }
p { color: var(--main-color); }
```
`.theme-a` selectors are ignored, that's fine. Element `<p>` stays red ✔️ 

But it doesn't work with attribute selectors: https://codepen.io/nseni/pen/zYGBEqJ
```css
:root { --main-color: red; }
:root[data-theme="a"] { --main-color: green; }
:root[data-theme="b"] { --main-color: blue; }
p { color: var(--main-color); }
```
Element `<p>` becomes blue instead of keeping the default color ❌

Can this be fixed?

**Comment by jhildenbiddle** [comment]
Thanks for filing the issue, @nseni.

A fix for this issue will be published soon (likely tomorrow).

**Comment by jhildenbiddle** [comment]
Fixed in 2.2.1
